src/components/savedLocations/components/SavedLocations.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { PageHeader, List, Tooltip, Button, Tag, Typography, Empty } from 'antd';
+import {
+    LOCATION_NUMBER_LABEL, DESCRIPTION_CREATE, CREATE_NOW,
+    EMPTY_ICON_URL
+} from '../constants';
+
+const SavedLocations = ({ history }) => {
+    const listData = [];
+    const viewSavedLocation = savedLocationId => {
+        console.log(savedLocationId);
```

Comment:
remove console.logs

Reply:
Okay. Though i just created the method there. to implement in the next pr.